akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -200,7 +203,6 @@ private[akka] class RemoteActorRefProvider(
         local.registerExtraNames(Map(("remote", d)))
         d
       },
-      serialization = SerializationExtension(system),
```

Comment:
I know that wrong configuration of serializers will fail the startup of the ActorSystem, by throwning before returning from apply/create. That is very nice. E.g. wrong class name in the bindings. I wonder if that was triggered by this? If that was the case we should initialize the SerializationExtension at a good place in the ordinary ActorSystem startup, probably right after the provider.

You can try with a plain local actor system with wrong serialization config and see if it throws.

Reply:
Good point, will try.